port/linux/omrvmem.c

```diff
@@ -468,9 +468,9 @@ findAvailableMemoryBlockNoMalloc(struct OMRPortLibrary *portLibrary, ADDRESS sta
 						addressRange_Init(&currentMmapRange, (ADDRESS)(uintptr_t)(-1), (ADDRESS)(uintptr_t)(-1));
 					} else {
 						char *next = NULL;
-						uintptr_t start = 0;
-						uintptr_t end = 0;
-						start = (uintptr_t)strtoull(lineBuf, &next, 16);
+						unsigned long long start = 0;
+						unsigned long long  end = 0;
```

Comment:
Should `unsigned long long int` be used as per http://www.cplusplus.com/reference/cstdlib/strtoull?
```suggestion
						unsigned long long int start = 0;
						unsigned long long int end = 0;
```

Reply:
The two are equivalent (there's an implicit `int` in `unsigned long long`, so it doesn't technically need to be specified).